app/web/features/notifications/locales/en.json

```
@@ -14,6 +14,8 @@
       "title": "Notification Settings",
       "description": "You may still receive important notifications about your account and content outside of your preferred notification settings.",
```

Can I make a cheeky unrelated change her to help with clarity?

```suggestion
      "description": "Any changes you make below are saved automatically. Please note that you may still receive important notifications about your account and content outside of your preferred notification settings.",
```